014-quarkus-panache-with-transactions-xa/src/test/java/io/quarkus/qe/containers/MysqlTestProfile.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.qe.containers;
+
+import java.util.Collections;
+import java.util.List;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class MysqlTestProfile implements QuarkusTestProfile {
```

Comment:
QuarkusTestProfile is not compatible when running the test on Native, so we're missing this coverage on Native. Do you think this coverage is not important for Native, so we can live without it?

Reply:
is not intended to be a native test.